src/main/webapp/WEB-INF/deployerConfigContext.xml

```diff
@@ -54,12 +51,106 @@
           c:authenticator-ref="authenticator">
         <property name="principalAttributeMap">
             <map>
-                <entry key="relayGuid" value="id" />
+            </map>
+        </property>
+    </bean>
+
+    <bean id="attributeRepository"
+          class="org.jasig.services.persondir.support.ldap.LdaptivePersonAttributeDao"
+          p:connectionFactory-ref="ldap.connection.factory.pooled"
+          p:baseDN="ou=sso,ou=account,dc=ccci,dc=org"
+          p:searchControls-ref="searchControls"
+          p:searchFilter="cn={0}">
+        <property name="resultAttributeMapping">
+            <map>
+                <!--
+                   | Key is LDAP attribute name, value is principal attribute name.
+                   -->
+                <entry key="relayGuid" value="ssoGuid" />
                 <entry key="cn" value="email" />
+                <entry key="givenName" value="first" />
+                <entry key="sn" value="last" />
             </map>
         </property>
     </bean>
 
+    <bean id="searchControls"
+          class="javax.naming.directory.SearchControls"
+          p:searchScope="1"
+          p:countLimit="2" />
+
+    <!-- start ldap connection pool configuration -->
+
+    <bean id="ldap.connection.factory.pooled" class="org.ldaptive.pool.PooledConnectionFactory"
+          p:connectionPool-ref="ldap.connection.pool" />
+
+    <bean id="ldap.connection.pool.bind" parent="ldap.connection.pool"
+          p:connectionFactory-ref="ldap.connection.factory.bind" />
+
+    <bean id="ldap.connection.factory.bind" parent="ldap.connection.factory">
```

Comment:
I don't think you need the `ldap.connection.pool.bind` and `ldap.connection.factory.bind` ldap beans that were defined in The Key. These should have been replaced by the `ldaptive:direct-authenticator`